exercises/binary_converter.cpp

```diff
@@ -5,3 +5,34 @@
   Insert first number: 8
   The binary number is: 1000
 */
+
+#include <stdlib.h>
+#include <stdint.h>
+#include <stdio.h>
+
+char* byte_to_string(uint8_t);
+
+int main(int argc, char **argv){
+	//Parameter check.
+	if(argc != 2){
+		printf("usage: %s <decimal-number>\n", argv[0]);
+		return 1;
+	}
+
+	//Main algorithm.
+	printf("%s\n", byte_to_string(atoi(argv[1])));
+
+	return 0;
```

Comment:
potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```